portfolio.html

```diff
@@ -0,0 +1,129 @@
+          </li>
+          <li class="categories__item">
+            <button class="categories__button">
+              Дизайн
+            </button>
+          </li>
+          <li class="categories__item">
+            <button class="categories__button">
+              Маркетинг
+            </button>
+          </li>
+        </ul>
+        <ul class="portfolio__projects projects">
+          <li class="projects__item project">
+            <img src="./images/project--1.jpg" alt="Project foto" class="project__image">
+            <div class="project__content">
+              <h3 class="project__title">Технокряк</h3>
+              <p class="project__category">Веб-сайт</p>
+            </div>
+          </li>
+          <li class="projects__item project">
+            <img src="./images/project--2.jpg" alt="Project foto" class="project__image">
+            <h3 class="project__title">Постер New Orlean vs Golden Star</h3>
+            <p class="project__category">Дизайн</p>
+          </li>
+          <li class="projects__item project">
+            <img src="./images/project--3.jpg" alt="Project foto" class="project__image">
+            <h3 class="project__title">Ресторан Seafood</h3>
+            <p class="project__category">Додаток</p>
+          </li>
+          <li class="projects__item project">
+            <img src="./images/project--4.jpg" alt="Project foto" class="project__image">
+            <h3 class="project__title">Проєкт Prime</h3>
+            <p class="project__category">Маркетинг</p>
+          </li>
+          <li class="projects__item project">
+            <img src="./images/project--5.jpg" alt="Project foto" class="project__image">
+            <h3 class="project__title">Проєкт Boxes</h3>
+            <p class="project__category">Додаток</p>
+          </li>
+          <li class="projects__item project">
+            <img src="./images/project--6.jpg" alt="Project foto" class="project__image">
+            <h3 class="project__title">Inspiration has no Borders</h3>
+            <p class="project__category">Веб-сайт</p>
+          </li>
+          <li class="projects__item project">
+            <img src="./images/project--7.jpg" alt="Project foto" class="project__image">
+            <h3 class="project__title">Видання Limited Edition</h3>
+            <p class="project__category">Дизайн</p>
+          </li>
+          <li class="projects__item project">
+            <img src="./images/project--8.jpg" alt="Project foto" class="project__image">
+            <h3 class="project__title">Проєкт LAB</h3>
+            <p class="project__category">Маркетинг</p>
+          </li>
+          <li class="projects__item project">
+            <img src="./images/project--9.jpg" alt="Project foto" class="project__image">
+            <h3 class="project__title">Growing Business</h3>
+            <p class="project__category">Додаток</p>
+          </li>
+        </ul>
+      </section>
+    </main>
+    <footer class="footer">
```

Comment:
add 1 empty line between adjacent closing / opening tags.

This will improve the readability of your code